static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -349,6 +349,26 @@ $block-height: 58px;
     }
 }
 
+// reserve space for comment count on commentable card
+.fc-item--is-commentable:not(.fc-item--type-comment) .fc-item__meta {
+    min-height: 17px; // 14px height + 3px margin-top of inline-icon
+    min-width: calc(3ch + 14px); // 14px height of inline-icon
```

Comment:
@sndrs I've removed the magic numbers used for the min-width with `3ch` which equals the width of `000`, plus 14px which is the width of the `inline-icon`.